### PR TITLE
Create common-annotations-1.3.adoc

### PR DESCRIPTION
**Pull request by ctomc**
https://issues.jboss.org/browse/WFLY-6463

### CODE REVIEW

File: ee/common-annotations-1.3.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+* WildFly
+* Weld Integration
+
+== Requirements
+
+Add conditional support for Common Annotations 1.3 This will be controlled by the `ee8.preview.mode` system property. If
+the property is not present or not set to `true` Comon Annotations 1.3 will be used instead.
+
+We need to check if Weld works fine with @Priority changes, which is tracked in https://issues.jboss.org/browse/WFLY-9748
```
Comment (mjurc) [inline ee/common-annotations-1.3.adoc]: I am not sure if this is correct. The Common Annotations 1.3[0] specification doesn't mention anything about `METHOD` level support for `@Priority`. Is this really required?

CCing @dmlloyd , @kabir 

[0] http://download.oracle.com/otn-pub/jcp/common_annotations-1_3-mrel3-spec/ chapter 2.7

File: ee/common-annotations-1.3.adoc [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+=== Affected Projects or Components
+
+* WildFly
+* Weld Integration
+
+== Requirements
+
+Add conditional support for Common Annotations 1.3 This will be controlled by the `ee8.preview.mode` system property. If
+the property is not present or not set to `true` Comon Annotations 1.3 will be used instead.
```
Comment (mjurc) [inline ee/common-annotations-1.3.adoc]: > If the property is not present or not set to `true` Comon Annotations 1.3 will be used instead.

I suppose this is a typo - just double checking. If EE8 is not enabled, Common Annotations 1.3 should NOT be present, or do I understand this wrong?

Comment (ctomc) [reply ee/common-annotations-1.3.adoc]: yeah it is a typo

File: ee/common-annotations-1.3.adoc (hunk 1 above)
Comment (ctomc) [reply ee/common-annotations-1.3.adoc]: see https://github.com/javaee/javax.annotation/commit/be897900de1042028dc2d5d3f67883537dc23a53#diff-2bcd13488e2c0c51fb8174b9235ef858R67

File: ee/common-annotations-1.3.adoc [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+
+Commons Annotations 1.3 is a small change update to 1.2 MR.
+Main change is allowing @Repeatble on @DatasourceDefintion and @Priority annotation is allowed to also on methods now.
```
Comment (mjurc) [inline ee/common-annotations-1.3.adoc]: `@Priority` is only for `ElementType.TYPE` and `ElementType.PARAMETER`. Perhaps remove the 'allowed on methods part.'

File: ee/common-annotations-1.3.adoc [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+=== Related Issues
+
+* https://issues.jboss.org/browse/WFLY-6463[WFLY-6463]
+* https://issues.jboss.org/browse/WFLY-9748[WFLY-9748]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
```
Comment (mjurc) [inline ee/common-annotations-1.3.adoc]: Add the contacts. Not sure if this is strictly required for umbrella issues like this, but feel free to add me as QE contact.

File: ee/common-annotations-1.3.adoc [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+* WildFly
+* Weld Integration
+
+== Requirements
+
+Add conditional support for Common Annotations 1.3 This will be controlled by the `ee8.preview.mode` system property. If
+the property is not present or not set to `true` Comon Annotations 1.2 will be used instead.
+
+We need to check if Weld works fine with @Priority changes, which is tracked in https://issues.jboss.org/browse/WFLY-9748
```
Comment (mjurc) [inline ee/common-annotations-1.3.adoc]: The issue is wrong. The correct issue is https://issues.jboss.org/browse/WELD-2239

File: ee/common-annotations-1.3.adoc (hunk 4 above)
Comment (ctomc) [reply ee/common-annotations-1.3.adoc]: added

**Pull request merged**